tests/src/CoreMangLib/cti/system/convert/ConvertToInt32_4.csproj

```diff
@@ -12,6 +12,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <CLRTestKind>BuildAndRun</CLRTestKind>
     <CLRTestPriority>1</CLRTestPriority>
+
+    <!-- Test unsupported outside of windows -->
```

Comment:
I don't think the comment is necessary here - the variable is self-explanatory.

Reply:
I understand what you are saying; however, I would like to keep it, as some tests have annotations with more specific information. Ideally there would be an exact reason with each test as to why it is unsupported, until then I consider the comment a place holder for the information.